pkgs/development/tools/npm-check/default.nix

```
@@ -0,0 +1,29 @@
+{ buildNpmPackage
+, fetchFromGitHub
+, lib
+}:
+
+buildNpmPackage rec {
+  pname = "npm-check";
+  version = "6.0.1";
+
+  src = fetchFromGitHub {
+    owner = "dylang";
+    repo = pname;
```

```suggestion
    repo = "npm-check";
```
Since `repo` and `pname` are semantically different, this seems to be preferred recently.